tests/integration/test_cloudformation.py

```diff
@@ -2405,3 +2405,30 @@ def test_cfn_with_multiple_route_tables(self):
 
         # Clean up
         self.cleanup(stack_name)
+
+    def test_cfn_with_multiple_route_table_associations(self):
+        ec2_client = aws_stack.connect_to_service("ec2")
+
+        resp = ec2_client.describe_vpcs()
+        vpcs_before = [vpc["VpcId"] for vpc in resp["Vpcs"]]
+
+        template = load_file(os.path.join(THIS_FOLDER, "templates", "template37.yaml"))
+
+        stack_name = "stack-%s" % short_uid()
+        create_and_await_stack(StackName=stack_name, TemplateBody=template)
+        resp = ec2_client.describe_vpcs()
+        vpcs = [vpc["VpcId"] for vpc in resp["Vpcs"] if vpc["VpcId"] not in vpcs_before]
+        self.assertEqual(1, len(vpcs))
```

Comment:
This might lead to concurrency issues. There could be another VPC created during the test which would make this assertion fail. The preferred way would be to use stack outputs here instead to return the created VPC id.

For the PR though, it's fine if we leave this as is for now.

Reply:
Now using template output for route table id, which is use in describe_route_tables as filter